### ISSUE TITLE
Documentation for deprecated items

### ISSUE THREAD

**Issue by RobinD42** [wxWidgets/Phoenix#748]
**Operating system**: any
**wxPython version**: any
**Stock or custom build**: any
**Python version**: any
**Stock or custom build**: any

**Description of the problem**:

Some (most?) of the @depreacted tags in the wxWidgets interface docs do not indicate the version when the item was deprecated, and it appears we are blindly substituting the current wxPython version into the documentation items. This is not accurate in almost all cases, and will cause confusion.

Track down the code that is doing that and change it to leave out the version and just say that it was deprecated. It's probably using a ReST directive and if the version is not optional there then something creative may need to be done.

**Comment by mesalu** [comment]
Looks like some creativity will be needed,
From the reST generated by build.py 
```reStructuredText
.. deprecated:: 4.1.0
         This function is deprecated in favour of :meth:`GetItemLabelText` .   
```
http://www.sphinx-doc.org/en/stable/markup/para.html#directive-deprecated

Should I fix #458 while I'm rewriting directives? ;)

**Comment by RobinD42** [comment]
Fixed in #762
